pkg/env/certificateRefresher_test.go

```diff
@@ -0,0 +1,283 @@
+AtEaeEplGG4AamtF4s35axIF8/jgMvkswHjodb2f79iImOx96tE/Odhu3T/uoR1V
+543Q1f80/HZzdpqnIpNZd/NrDMv2Hjm1LcLFMpcZFBpdF4CmAm6J59LjfZWm5qq5
+aJf3MFEdxsX0VwEURmP0lrsvZTJhsu+httv+soFareiOk2W5nU0UhFSpKfOrFfld
+cHxNPX6T2wr7JTYLDfgeRG6VoboPwcdVCCllrKWUlP9zdq+A6zbByHqC+QG6IP2c
+zzXUAMR9AgMBAAECggEAOHAjSpH7a+NzsH5fL88bakC87VhVy8IAMMYzGUysWbe6
+IhQj8lHqXdSmC8f8okgb5ooNNu2bpgUDpSxNmIikB4UiZ3fQsm2vM65V1d7rwy52
+V2DodIpKqIoBIPjb3v25IY0ZipHFP8v8epJgUPcTm0Y9lJgPXnqRAQzw0Gs319FI
+wxVy/RkwJ/p6BYufJs/T7HjCjCSsBA0JPx46qiCGA90u4gmYOwQFmZoEjlcMzPaL
+Oinbfs2WN5mSazev/4zY/bEnFuO1vbpvx1qAEZpoDvN2CeYjAHvhmMe5zQwEFgsY
+Bjhfd8pHuujbkIvHNo8mcRY8D7tFggw0QC8YZBl9+QKBgQDNcJQPQPaLUy8ZUxfL
+8HMfpnVPAt9g0mctMo52CiF95WqMjXhJ5tbn7J2R6JFawCoIunNAWDwCkOg9dzgE
+SwkGAQjSvHNlLOAIx1ej+vjeLvjyCgXqiv66bG16JK4ZA3K/Dewe/o27Tbpny0/7
+AkfjrknhyRjK4pnnZ/kWj6wJtwKBgQD3qqrXHBJD78lo0HnZmNLeI8noM5ruJD23
+9dfTEkr99/cpe6mdD2OJKNsBfUB4y/Ces7EHPa6UEPLl36vIHhg+5chrqrNbRfIv
+LeCeRPkjVTxEuPCtUx7Bt1LqlM1tPfOp67V6160aroLnpIdUsWSOQFuAvGqHKLhG
+Nau31SDzawKBgQC9PIYlxuFTVTx9R10ULljdPqewMCUzOpxvtbIkaRCQt1J+RZIY
+ANrUp9A9Js09muUdRSIEk0Iz2ucSN08SJUwai7lk5NIm0D9N1tGT6wpzHzGRQkpQ
+0dfyQQ5XBJKZ1+NKubhWlIRZlC+wjEcQH/m4cEL+CA8eU70Qu2VmstD14QKBgEmo
+PWT6WUhRMUJ19jdL5zLfy/W+G07GAoEKoaSJpToBHEX/HEO0xvKM7w1zVdBXPvnE
+EVtI8fnhTIwnSGyc3rMeHcw/mVYE6HE1oL8RXlMuz1zU7+dseBI+1m8j0DC0Ixqf
+GnstV7M+wXnpCcKbe39/DnesEbae2qcu4SIsRb9/AoGBAJw6iTZm2kjk/AK3i/ZN
+RpuP6/8jbHZNTvLz8Aw9J4wxoM8wp3kpJe0kMOgzWZPiya4QX7kCma+uoibiR3v/
+ohPVpYqZtLFdfEd9pDXMZCg/q1sOcL94gNiv9w0QCbuvBhAno7OHk9JZMIr3pJ0h
+tT1hverLghBfQ8KhO9Gk+FSM
+-----END PRIVATE KEY-----
+-----BEGIN CERTIFICATE-----
+MIIDLjCCAhagAwIBAgIQD/cl3WNvT3ynxkUONemaPjANBgkqhkiG9w0BAQsFADAU
+MRIwEAYDVQQDEwlwZXRyLnRlc3QwHhcNMjEwNTI2MDkzNDQzWhcNMjEwNjI2MDk0
+NDQzWjAUMRIwEAYDVQQDEwlwZXRyLnRlc3QwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+DwAwggEKAoIBAQDGwJS2bDR2ugfsODr1/VkhX9A2b6/w66r4habosbIeX9meMsKZ
+btsMOrWH2uten9H7L0o2rYbU5c3LAtEaeEplGG4AamtF4s35axIF8/jgMvkswHjo
+db2f79iImOx96tE/Odhu3T/uoR1V543Q1f80/HZzdpqnIpNZd/NrDMv2Hjm1LcLF
+MpcZFBpdF4CmAm6J59LjfZWm5qq5aJf3MFEdxsX0VwEURmP0lrsvZTJhsu+httv+
+soFareiOk2W5nU0UhFSpKfOrFfldcHxNPX6T2wr7JTYLDfgeRG6VoboPwcdVCCll
+rKWUlP9zdq+A6zbByHqC+QG6IP2czzXUAMR9AgMBAAGjfDB6MA4GA1UdDwEB/wQE
+AwIFoDAJBgNVHRMEAjAAMB0GA1UdJQQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjAf
+BgNVHSMEGDAWgBTVoGOzKcn+Y+vypnh2e5BzIcQ6NzAdBgNVHQ4EFgQU1aBjsynJ
+/mPr8qZ4dnuQcyHEOjcwDQYJKoZIhvcNAQELBQADggEBAGREAWRpsAUlhLtnqgyh
+qhRDd0xuzlBte5uMWYJxojwK7jmJlb4YbIgm6OfSMW3cd1xN497egji5AiiW37wN
+UDONqYUL4qF3nnPJl9yjvptJCdpZQC1Ma5b2BLF8X8kurh8rYtyx2NNncj6/p/IA
+D/R4qcuA0UaXLdARsppjLTCzk6T/Z8v4qaaXQJBfdHLY2fZo4vSI1yLyTcGia703
+Zn30eMz0PLFCkMEYhZADUia0peReH08WO7RTtE5Nf8r5ZiFSfuJJlBmURMU3MAxJ
+PeihXgcD511I3eMq7A8r7+xlNFEFYaleYwmoYSOt0cBGtoZttXDYacR/xvEkVaxO
+1LQ=
+-----END CERTIFICATE-----`
+
+// equalPrivKey is added manually, it has been added to go1.15, whilst ARO uses go1.14
+// TODO: Remove once ARO is on go1.15
+func equalPrivKey(one, other *rsa.PrivateKey) bool {
```

Comment:
The public and private equal functions are added by go1.15. I needed to copy this one to simplify testing.